src/main/java/com/recurse/portfolio/data/DisplayAuthor.java

```diff
@@ -41,23 +44,30 @@ private static DisplayAuthor anonymous() {
         return new DisplayAuthor(
             0,
             "Anonymous",
-            "/user-placeholder.png"
+            "/user-placeholder.png",
+            null
         );
     }
 
     private static DisplayAuthor usePublic(User user) {
+        String slug = slugify.slugify(user.getPublicName());
+
         return new DisplayAuthor(
             user.getUserId(),
             user.getPublicName(),
-            user.getPublicImageUrl()
+            user.getPublicImageUrl(),
+            slug
         );
     }
 
     private static DisplayAuthor useInternal(User user) {
+        String slug = slugify.slugify(user.getInternalName());
```

Comment:
I think we had talked about using the public name throughout for profile slugs, regardless of whether or not the viewer is logged in... or at least, that's what I [wrote down](https://github.com/jasonaowen/recurse-community-portfolio/issues/12#issuecomment-501794259) after our discussion! I'm fine with going with this instead, but I just wanted to double-check that it was intentional?

Reply:
Yes, this was intentional.  We decided to use internal info for a user's pages when they are logged in.  So I'm good with this, I guess we just need a confirmation from @EL246. :)

Reply:
Thanks, @j9peters! I'm not sure we need to block on @EL246 responding, but I'll wait until tomorrow before merging.